### ISSUE TITLE
iframe and microphone permission

### ISSUE THREAD

**Issue by tom2bec** [zendesk/sunshine-conversations-web#681]
**I'm submitting a ...**
  - [x] bug report
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

 **What is the current behavior?**
when I push a `webview` message to smootch web messenger, `webview` is opening in the customer side with uri set up  but I can't get microphone permission granted to this page.
```
smooch.appUsers.sendMessage('xxxxxxxx','yyyyyyy', {
           role: 'appMaker',
           type: 'text',
           text: 'snapcall',
           actions: [
               {
                   type: 'webview',
                   text: 'make a call',
                   size: 'tall',
                   uri: 'https://snapcall.io/button',
                   fallback: 'https://smootch.io'
               }
           ]
       }
   );
```

 **What is the expected behavior?**
Need  to set up parameter `allow=microphone,camera` in web messenger iframe declaration

example :
```
<iframe width="90%" height="500px" src="//appr.tc" allow="microphone;camera"></iframe>
```

 **Please tell us about your environment:**
- Delivery method: [CDN]
- Version (can be found using `Smooch.VERSION` in your browser): 4.25.4
- Browser: [all]
- OS: [all]

 **Other information** 
I'm from [SnapCall](https://snapcall.io) and we try to embed in a webview message type our `WebRTC` direct call button.

**Comment by DataWearsAHood** [comment]
Hi Tom,

If I've understood correctly, even with the `allow="microphone;camera"`, you're still not able to access the microphone.
The dev team has confirmed this is not possible ATM, but I've logged your feedback/interest in such a feature with the product team.

**Comment by tom2bec** [comment]
Hey Adam,

thx for your feedback !

Thomas de Becdelièvre
SnapCall
Co-founder
+33679716293
https://snapcall.io

**Comment by jpjoyal** [comment]
@tom2bec This should now be fixed in [5.2.2](https://github.com/zendesk/sunshine-conversations-web/releases/tag/5.2.2) Let us know if that works and that can now run Snapcall in Web Messenger's webview.

**Comment by tom2bec** [comment]
@jpjoyal thx it works perfectly for audio and video 👍 


https://user-images.githubusercontent.com/22027434/106019840-625bba80-60c3-11eb-9850-e89ff492bce1.mov
